### DISCUSSION TITLE
Serial dependence between 2 dags

### DISCUSSION THREAD

**Discussion by DimitrisSotiriou** [argoproj/argo-workflows#6941] [General]
Not sure if the title describes very well my need, but here is more context. I have 2 dags run successively if the second dag fail I want the previous to rerun in case of retry. Is that possible?

**Comment by sarabala1979** [comment]
yes you can run. Please refer to the coin-flip examplehttps://github.com/argoproj/argo-workflows/blob/master/examples/coinflip-recursive.yaml. you check the  DAG2 condition and invoke  DAG1.
